src/octoprint/plugins/events/static/js/events.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * View model for OctoPrint-EventCoordinator
+ *
+ * Author: jneilliii
+ * License: MIT
+ */
+$(function () {
+    function EventcoordinatorViewModel(parameters) {
+        var self = this;
+
+        self.settingsViewModel = parameters[0];
+        self.selected_command = ko.observable();
+
+        self.event_details = function (data) {
```

Comment:
Best try to avoid mixed camelCase and snake_case. I know that I'm a culprit there too, but I try to avoid adding more and fix stuff that I see.